CMakeLists.txt

```diff
@@ -1370,7 +1239,7 @@ set(
 
 set(
   IRODS_MAIN_EXECUTABLES
-#  irodsAgent
+# irodsAgent
```

Comment:
is this cruft?

Reply:
This is cruft from the past. I just adjusted the alignment.

Reply:
torch it

Reply:
removed it.